epoch/src/lib.rs

```diff
@@ -0,0 +1,141 @@
+use solana_account_decoder::parse_sysvar::SysvarAccountType;
+use solana_client::client_error::ClientError;
+use solana_rpc_client::nonblocking::rpc_client::RpcClient;
+use solana_sdk::epoch_info::EpochInfo;
+use solana_sdk::slot_history::Slot;
+use solana_sdk::sysvar::epoch_schedule::EpochSchedule;
+use std::sync::Arc;
+
+#[derive(Debug, Copy, Clone, PartialOrd, PartialEq, Eq, Ord)]
+pub struct Epoch {
+    pub epoch: u64,
+    pub slot_index: u64,
```

Comment:
I do not see why we need to create a new library for epoch, this file can go in core library.

Reply:
I didn't know where to put it so as it's independent, I put in a crate until we decide where to put.
I put it in core/structures?